doc/01_intro.md

```diff
@@ -0,0 +1,37 @@
+# Introduction {#mainpage} #
+The purpose of a [synchronization client](@ref synchronization_client) for an AsteroidOS watch is to provide a means by which data and commands from the machine, generally, a computer or mobile phone running Linux on which the client runs, to the watch.  Although either a computer or mobile phone are commonly used for this, the rest of this document will refer to "the computer" to mean the computer on which the synchronization client runs.
+
+It's important to understand that this current work only provides *half* of the required software, and that a separate synchronization client then uses D-Bus to actually perform the synchronization.
```

Comment:
This _separate synchronization client_ runs of the watch right?

Reply:
No, it runs on the computer.  That is really a terrible couple of paragraphs which I have now rewritten.